types/react-linkify/index.d.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as React from "react";
 
-declare namespace ReactLinkify {
+export namespace ReactLinkify {
```

Comment:
would you mind sharing how you plan on extending such, can it be achieved with current decorator overrides?

Reply:
I basically need access to the props so that I can build components that extend the original functionality. Something like...
```tsx
import * as React from 'react';
import ReactLinkify, { ReactLinkify as RL } from 'react-linkify';

export const Linkify: React.FC<RL.Props> = (props) => (
  <ReactLinkify componentDecorator={componentDecorator} {...props}>
    {props.children}
  </ReactLinkify>
);

const componentDecorator = (href: string, text: string) => (
  <a href={href} target="_blank">
    {text}
  </a>
);
```